blocks/image-link-slider/image-link-slider.js

```diff
@@ -91,34 +105,23 @@ export default async function decorate(block) {
       const carouselItems = document.createElement('div');
       carouselItems.classList.add('carousel-items');
       const apiName = configNameElement.nextElementSibling.textContent.trim();
-      callAPI(apiName)
-        .then((data) => {
+      const formData = new FormData();
+      if (apiName === 'finace') {
+        formData.append('apiName', 'GetFinaceListing');
+      }
+      formData.append('inputJson', JSON.stringify({ pageNo: '1', pageSize: '5' }));
+      // eslint-disable-next-line consistent-return
+      postFormData(getResearchAPIUrl(), formData, async (error, apiResponse = []) => {
+        if (apiResponse) {
+          const jsonResult = parseResponse(apiResponse);
           if (block.classList.contains('image-link-slider')) {
-            renderImageLinkVariant(data, carouselItems, block?.dataset?.maxLimit);
-          }
-          return loadCarousel(block, carouselItems);
-        })
-        .then(() => {
-          block.querySelectorAll('.carousel-slide').forEach((slide) => {
```

Comment:
@ravverma, imho it wasn't a good idea to take out this optimisation. `data-src` would prevent fetching all the images on page load. Instead, they would  be fetched one by one when the user clicks next on the Carousel.